app/views/people/tasks.html.erb

```diff
@@ -0,0 +1,24 @@
+<h1><%= "#{@person.name}'s Tasks" %></h1>
```

Comment:
When I look at this page I don't see any people that have tasks associated with them. Some investigation may be needed to see why the list of tasks is empty.